src/_data/data.js

```diff
@@ -1,250 +1,283 @@
-    permalink: `/pending-guidance/${filename.replace('.md', '')}/`
   };
-}
-
-// Cross-reference enrichment
-function enrichWithGuidance(faqItems, guidanceItems) {
-  return faqItems.map(faqItem => {
-    const hasGuidanceId = Boolean(faqItem['guidance-id']);
-
-    // Find related guidance document if guidance-id exists
-    const guidanceItem = hasGuidanceId
-      ? guidanceItems.find(guidance => guidance.filename === faqItem['guidance-id'] + '.md') || null
-      : null;
-
-    // Determine danger indicators
-    const guidanceFileNotFound = hasGuidanceId && !guidanceItem;
-
-    // Throw error if guidance ID is missing for pending-guidance status
-    if (faqItem.status === 'pending-guidance' && !hasGuidanceId) {
-      throw new Error(
-        `Missing guidance-id for FAQ with status 'pending-guidance':\n` +
-        `  Permalink: ${faqItem.permalink}\n`
-      );
-    }
+};
 
-    return {
-      ...faqItem,
-      guidanceItem,
-      guidanceFileNotFound
-    };
+// Process all guidance request files into structured objects
+function createProcessedGuidanceRequests(guidanceDir) {
+  const guidanceFiles = getGuidanceFiles(guidanceDir);
+  const guidanceRequests = getParsedMarkdownFiles(guidanceFiles);
+  const processedGuidanceRequests = guidanceRequests.map(guidanceRequest => {
+    return getProcessedGuidanceRequest(guidanceRequest);
   });
-}
 
-function enrichWithRelatedFaqs(guidanceItems, faqItems) {
-  return guidanceItems.map(guidance => {
-    // Find all related FAQs that reference this guidance
-    const relatedFaqs = [];
-    const guidanceKey = guidance.filename.replace('.md', '');
-
-    for (const faqItem of faqItems) {
-      if (faqItem['guidance-id'] === guidanceKey) {
-        relatedFaqs.push({
-          category: faqItem.category,
-          filename: faqItem.filename,
-          question: faqItem.question,
-          permalink: faqItem.permalink
-        });
-      }
-    }
+  return processedGuidanceRequests;
+};
 
-    return {
-      ...guidance,
-      relatedFaqs,
-      relatedFaq: relatedFaqs[0] || null // Keep for backward compatibility
-    };
+// ============================================================================
+// Curated List Processing
+// ============================================================================
+
+// Get README.yml files from FAQ subdirectories
+function getCuratedListFiles(faqDir) {
+  const files = fs.readdirSync(faqDir, { withFileTypes: true, recursive: true });
+
+  const curatedListFiles = files.filter(entry => {
+    return entry.isFile() &&                    // Only files
+      entry.name === 'README.yml' &&            // Must be named README.yml
+      entry.parentPath !== faqDir;              // Not at the root of FAQ directory
   });
+
+  return curatedListFiles;
 }
 
-// Data organization
-function organizeFaqsByCategory(faqItems) {
-  const result = {};
+// Parse a curated list
+function getProcessedCuratedList(curatedList) {
```

Comment:
Nit: `get` prefix implies somehow fetching this or at the very least getting a property from somewhere. Not super sure what the best name is, here. Not a blocker, just food for thought.